vspace/src/main/webapp/WEB-INF/views/staff/images/imagelist.html

```diff
@@ -1,204 +1,243 @@
+                        th:id="'category-badge-'+${image.id}+'-'+${cat}"
+                        th:data-category="${cat}"
+                        class="badge badge-warning">[[${@environment.getProperty('image_category_'
+                     + cat)}]] <i
+                        th:data-image-id="${image.id}"
+                        th:data-category="${cat}"
+                        class="fas fa-times removeTag"></i>
+                     </span>
+                     </span>
+                     <form id="changeTagForm"
+                        th:action="@{'/staff/images/'+${image.id}+'/tag?'+${_csrf.parameterName}+'='+${_csrf.token}}"
+                        method="post">
+                        <input type="hidden" name="imageID"
+                           th:value="${image.id}" id="imageID" />
+                        <select id="changeTag" name="tag"
+                           th:onChange="toggleChange(this.form, '[(${image.id})]')"
+                           class="form-control form-control-sm"
+                           style="width: 68px;">
+                           <option>Assign a tag</option>
+                           <option
+                              th:each="catTag:${imageCategories}"
+                              th:value="${catTag}">[[${@environment.getProperty('image_category_'
+                              + catTag)}]]
+                           </option>
+                        </select>
+                     </form>
+                  </td>
+                  <td style="display:none;" class="filteredData"><span
+                     th:each="cat:${image.categories}">[[${cat}]]
+                     </span>
+                  </td>
+               </tr>
             </tbody>
-        </table>
-        <div id="page-selection"></div>
-    </div>
-</body>
+         </table>
+         <div id="page-selection"></div>
+         <script>
+            $(document).on('input', '#searchImageField', function(e){
+             
+             if($("#searchImageField").val().trim()) {
+            	 
+                     $('#searchImageButton').removeAttr('disabled');
+             }
+             else {
+                    $('#searchImageButton').attr('disabled','true');
+             }
+             if (!$('#searchImageButton').attr('disabled') && $("#searchImageField").val().trim()){
+            	 
+            		$("#searchImageLink").attr('href','[[@{/staff/images/search/full/1?searchText=}]]'+$("#searchImageField").val());
+            		
+             } 
+             else{
+            	 $("#searchImageLink").attr('href','');
+             }
+            });
+            $(document).on('keydown', function(e) { 
+            
+                if((e.key  === 'Enter' || e.keyCode === 13) && $("#searchImageField").val().trim()) {
+               	 
+                    $("#searchImageButton").trigger('click');
+                }
+            }) 
+         </script>
+      </div>
+   </body>
```

Comment:
only the parts that are actually changed should be changed. The way it is right now, I can't see what you have actually change in this file.

Reply:
Took the version file from develop and added my code for this ticket again.